packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -36,14 +36,18 @@ const wrapFunctionForErrors = fn => async (...args) => {
   }
 };
 
-const createProvider = ({ provider, providerOptions }) => {
+const createProvider = ({ provider, providerOptions, actionOptions  }) => {
   try {
     const providerInstance = require(`strapi-provider-upload-${provider}`).init(providerOptions);
 
     return Object.assign(Object.create(baseProvider), {
       ...providerInstance,
-      upload: wrapFunctionForErrors(providerInstance.upload.bind(providerInstance)),
-      delete: wrapFunctionForErrors(providerInstance.delete.bind(providerInstance)),
+      upload(file, options = actionOptions.upload) {
```

Comment:
you will need to keep the wrapFunctionForErrors  arround ;)

Reply:
This should now be resolved